port.json

```
@@ -0,0 +1,3 @@
+{
+    "port": "3000"
+}
```

File ini dihapus aja atau di rename jadi `.env.sample` dan isinya nanti jadi begini

```suggestion
PORT=
```